### PR TITLE
parser: fix set upper-case variable

### PR DESCRIPTION
**Pull request by zimulala**

### CODE REVIEW

File: parser/lexer.go [hunk 1]
```diff
@@ -243,7 +243,7 @@ func startWithAt(s *Scanner) (tok int, pos Pos, lit string) {
 		tok, lit = userVar, s.r.data(&pos)
 	} else if ch1 == '@' {
 		s.r.inc()
-		stream := s.r.s[pos.Offset+2:]
+		stream := strings.ToLower(s.r.s[pos.Offset+2:])
```
Comment (tiancaiamao) [inline parser/lexer.go]: I suggest write a `hasPrefix` case insensitive function here, it has two advantages:

1. remain the raw input data as it is (error message should be benefited).
2. reduce object allocation.

**Comment by shenli** [comment]
LGTM

**Comment by tiancaiamao** [comment]
LGTM

**Comment by tiancaiamao** [comment]
LGTM

**Pull request merged**